src/main/java/views/monthly_menu/MonthMenuV.java

```diff
@@ -201,4 +200,13 @@ public void actionPerformed(ActionEvent event) {
             // call generateSummary
         }
     }
+
+    /**
+     * Open Month Menu and notify user.
+     * @param message notify user that Month Menu is updated
+     */
+    public void openMonthMenu(String message) {
```

Comment:
Do you think you could make the message optional, kinda like how I did it in openMainMenu? Because when I call this from the main menu I think having a popup every time you select a month would be kind of annoying to the user

Reply:
I'll remove the message completely. I don't think I could make it optional.

Reply:
Also, I just realized the line 
_System.out.println(session);_ 
I don't think I meant to write this. I'll remove it.

Reply:
The way I did it was that if you call the function with null for the message it will check for that and not do the popup

Reply:
But if you want to remove it completely then that's fine too. Whichever you prefer

Reply:
Actually I'll put it back. I made it the same way you did it.